src/features/amUI/common/OldRolesAlert/OldRolesAlert.tsx

```diff
@@ -0,0 +1,37 @@
+import { getHostUrl } from '@/resources/utils/pathUtils';
+import { useTranslation } from 'react-i18next';
+
+import { ExternalLinkIcon } from '@navikt/aksel-icons';
+import { Link } from 'react-router';
+import { Alert, Heading, Paragraph, Link as DsLink } from '@digdir/designsystemet-react';
+import styles from './OldRolesAlert.module.css';
+
+export const OldRolesAlert = () => {
+  const { t } = useTranslation();
+  return (
+    <Alert data-color='info'>
+      <div className={styles.container}>
+        <Heading
+          data-size='xs'
+          level={3}
+          style={{
+            marginBottom: 'var(--ds-size-2)',
+          }}
+        >
+          {t('a2Alerts.launchAlertHeading')}
+        </Heading>
+
+        <Paragraph>{t('a2Alerts.launchAlertContent')}</Paragraph>
+        <DsLink asChild>
+          <Link
+            to={`${getHostUrl()}ui/profile/`}
```

Comment:
As a courtesy to the user, I think we should redirect them directly to the rights-page of the user they are currently looking at, instead of just to the profile page. That would save them the trouble of having to find the same user in the A2-list of users and then opening their page.

This is something we already do in the old single-rights GUI, so you can actually get the url just using this function: `redirectToSevicesAvailableForUser` 😊